### PR TITLE
Added support for PlaceholderAPI 1.4^

### PR DESCRIPTION
**Pull request by MEFRREEX**

### CODE REVIEW

File: src/main/java/scoretag/APIDownloader.java [hunk 1]
```diff
@@ -18,7 +18,7 @@ public static void checkAndRun(Main plugin) {
 
             try {
                 FileOutputStream fos = new FileOutputStream(placeholderApi);
-                fos.getChannel().transferFrom(Channels.newChannel(new URL("https://dl.dropboxusercontent.com/s/6rmogms1458p369/KotlinLib.jar").openStream()), 0, Long.MAX_VALUE);
+                fos.getChannel().transferFrom(Channels.newChannel(new URL("https://cloudburstmc.org/resources/kotlinlib.48/version/2442/download").openStream()), 0, Long.MAX_VALUE);
```
Comment (PetteriM1) [inline src/main/java/scoretag/APIDownloader.java:21]: The Cloudburst website doesn't allow this

File: src/main/java/scoretag/TagUpdater.java [hunk 2]
```diff
@@ -16,10 +14,12 @@ public TagUpdater(Main plugin) {
     public void run() {
         try {
             for (Player p : plugin.getServer().getOnlinePlayers().values()) {
-                String tag = (String) plugin.placeholderAPI.getDeclaredMethod("translateString", String.class, Player.class).invoke(PlaceholderAPI.getInstance(),
-                        plugin.tag.replace("%factions_name%", plugin.getFaction(p)).replace("%device_os%", plugin.getOS(p)), p);
-                if (!p.getScoreTag().equals(tag)) {
-                    p.setScoreTag(tag);
+                String tag = (String) plugin.tag.replace("%factions_name%", plugin.getFaction(p)).replace("%device_os%", plugin.getOS(p));
+                if (Main.api != null) {
```
Comment (PetteriM1) [inline src/main/java/scoretag/TagUpdater.java:18]: Main.api should never be null

File: src/main/java/scoretag/APIDownloader.java (hunk 1 above)
Comment (MEFRREEX) [reply src/main/java/scoretag/APIDownloader.java:21]: It works

Comment (MEFRREEX) [reply src/main/java/scoretag/APIDownloader.java:21]: You can see for yourself

File: src/main/java/scoretag/TagUpdater.java (hunk 2 above)
Comment (MEFRREEX) [reply src/main/java/scoretag/TagUpdater.java:18]: I took LuckChat as an example to update the plugin 
https://github.com/ZenSirzechs/LuckChat-NK/blob/c6c173df1a577db25936ab2c9ba65d3b215e37e2/src/main/java/zen/luckchat/NameTag.java#L36

File: src/main/java/scoretag/APIDownloader.java (hunk 1 above)
Comment (PetteriM1) [reply src/main/java/scoretag/APIDownloader.java:21]: "Server returned HTTP response code: 403 for URL: https://cloudburstmc.org/resources/kotlinlib.48/version/2442/download"

**Comment by PetteriM1** [comment]
Fixed in 865842c

**Pull request closed without merging**